### DISCUSSION TITLE
Zigbee2MQTT LXC installation error

### DISCUSSION THREAD

**Discussion by donnyyu** [tteck/Proxmox#299] [Q&A] [answered]
Hi Guys, anyone had this error running `bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/zigbee2mqtt-v3.sh)"` ?

i got this error:
`
  _______       _               ___  __  __  ____ _______ _______ 
 |___  (_)     | |             |__ \|  \/  |/ __ \__   __|__   __|
    / / _  __ _| |__   ___  ___   ) | \  / | |  | | | |     | |   
   / / | |/ _  |  _ \ / _ \/ _ \ / /| |\/| | |  | | | |     | |   
  / /__| | (_| | |_) |  __/  __// /_| |  | | |__| | | |     | |   
 /_____|_|\__, |____/ \___|\___|____|_|  |_|\___\_\ |_|     |_|   
        v3 __/ |                                                  
          |___/                                                   

‼ ERROR 1@68 Unknown failure occured.
‼ ERROR 1@68 Unknown failure occured.`
what does it mean?

I am running v6.2 proxmox.

cheers

**Comment by tteck** [comment]
It should have said "This script requires Proxmox Virtual Environment 7.0 or greater"

**Comment by tteck** [comment, marked as answer]
Line 68 in `https://github.com/tteck/Proxmox/blob/main/ct/zigbee2mqtt-v3.sh`
```
function PVE_CHECK() {
    PVE=$(pveversion | grep "pve-manager/7" | wc -l)

    if [[ $PVE != 1 ]]; then
        echo -e "${RD}This script requires Proxmox Virtual Environment 7.0 or greater${CL}"
        echo -e "Exiting..."
        sleep 2
        exit
    fi
}
```

**Comment by donnyyu** [comment]
@tteck Thank you for quick reply and great work, I will have a play v7
